.gitignore

```diff
@@ -41,3 +41,6 @@ etc/kayobe/inventory/group_vars/seed/ansible-host
 
 # Ignore kolla configuration.
 etc/kolla
+
+# Ignore ansible vault password
+.vault_password
```

Comment:
This isn't really a standard filename, but nvm.

Reply:
+1